tests/lava/proc/monitor/test_monitors.py

```diff
@@ -342,6 +342,141 @@ def test_proc_params_accessible_in_proc_model(self):
         self.assertEqual(next(iter(exe.py_builders)).proc_params,
                          monitor.proc_params)
 
+    # New API unittests
+    def test_single_recorder_with_single_targer(self):
+        """Check if a single Recorder can monitor the spike output of a
+        neuron"""
+        shape = (1,)
+        num_steps = 6
+        neuron = LIF(shape=shape,
+                     vth=3,
+                     bias=1)
+
+        recorder = Recorder()
+        recorder.probe(neuron.s_out)
```

Comment:
I'd still prefer 'monitor' as the method name. What do you think? I don't think we should stick to the "neuron" analogy with a probe and a cell as Lava is more general. I know Andreas likes "probe" better but maybe we can convince him.